core/opa_transport.go

```diff
@@ -40,26 +40,40 @@ type OPATransport struct {
 	request                  *http.Request
 	permission               *openapi.RondConfig
 	partialResultsEvaluators PartialResultsEvaluators
-	env                      config.EnvironmentVariables
+
+	clientHeaderKey         string
```

Comment:
can we wrap these options in a Options/RequestOptions struct?

Reply:
Done